PhotoKingdomAPI/PhotoKingdomAPI/Controllers/ResidentsController.cs

```diff
@@ -28,18 +28,16 @@ public IHttpActionResult Get(int? id)
             // Fetch the object, so that we can inspect its value
             var o = m.ResidentWithDetailsGetById(id.Value);
 
+            if(o == null)

```

Comment:
Place check for null up here because otherwise it would throw an exception if wrong ID passed in and we check for the object's ResidentAttractionOwns.Count() below